src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1027,7 +1027,7 @@ TxQ::apply(Application& app, OpenView& view,
             JLOG(j_.warn()) <<
                 "Removing last item of account " <<
                 lastRIter->account <<
-                "from queue with average fee of" <<
+                " from queue with average fee of " <<
```

Comment:
I don't think there should be a space at the end, right?

Reply:
Yeah, the next line of code starts with a number. This is what it looks like now (from https://github.com/ripple/rippled/issues/2215):
> 2017-Aug-28 22:47:24 TxQ:WRN Removing last item of account rfepxFxQKMSwn1otQRk35TgXbfLX6p8JBXfrom queue with average fee of27297 in favor of 10982E55CC5219B0B4B90B5B2D205965196171F8C8A720800EF14D6B9F789C21 with fee of 446796

Reply:
Oh, you're right, I misread the code.